exo-go/test_helpers/run_feature_context.go

```diff
@@ -40,6 +40,12 @@ func RunFeatureContext(s *godog.Suite) {
 		return CheckoutApp(cwd, appName)
 	})
 
+	s.Step(`^I am in the "([^"]*)" directory of the "([^"]*)" example application$`, func(serviceName, appDirName string) error {
+		appDir = path.Join(cwd, "tmp", appDirName, serviceName)
```

Comment:
This feels wrong since the appDir doesn't include the serviceName. I'd suggest using the existing step for picking what application you are in and creating a new step for something like
```
When starting "exo test" in the "exoservice" directory
```
Where the dir your run is would be appDir plus exoservice.